README.md

```diff
@@ -260,23 +126,7 @@ If you like this project...
 
 > Give credit to everyone who inspired your codebase.
```

Comment:
- [ ] Please remove this comment. It is from the default README and it is just provided information on what a student should write under this header. Please do the same for the "Show your support" header.